src/phoebus_utils/root_find.hpp

```diff
@@ -18,6 +18,44 @@ namespace root_find {
 
 enum class RootFindStatus { success, failure };
 
+template <typename F>
+KOKKOS_INLINE_FUNCTION void broyden2(F &func, Real v[2], const int max_iter = 50,
```

Comment:
Not fully implemented/tested but we would want something like this if we do 2D rootfinds over T/Ye for the radiation a la Fornax.

Reply:
:+1: If I recall, @lroberts36 has something like this in the moment code too right? Or is that a different-dimensional root find?

Reply:
I think that's an analytic inversion but maybe I'm mistaken.